corehq/apps/hqadmin/views.py

```diff
@@ -269,6 +270,21 @@ def system_ajax(request):
     return HttpResponse('{}', content_type='application/json')
 
 
+@require_superuser_or_developer
+def test_blobdb(request):
+    """Save something to the blobdb and try reading it back."""
+    db = get_blob_db()
+    bucket = "test_blobdb_{}".format(datetime.now().toordinal())
+    contents = "It takes Pluto 248 Earth years to complete one orbit!"
+    info = db.put(StringIO(contents), bucket)
```

Comment:
I think this won't work. `bucket` is going in as the `basename` parameter, and later when you try to delete it will fail because this blob will be in the "default" bucket. Probably better to omit the `basename` and `bucket` parameters (on this line and the `db.delete(...)` line); the defaults are fine in this case.